### PR TITLE
Feature/SK-974 | Add example notebook for tuning of server side hyperparameters

### PR DESCRIPTION
**Pull request by benjaminastrand**
* First draft of notebook added
* Minor changes in documentation for Python API

### CODE REVIEW

**Comment by benjaminastrand** [comment]
* Cleaned up code
* Rewrote all markdown blocks
* Ran 20 trials à 50 rounds -> Nice final plot of objective function and learning rate

**Review by Wrede — APPROVED** [review]
Very good!
